modules/admin/proc-rhdh-deployment-config.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in:
+// title-admin.adoc
+
+[id="proc-rhdh-deployment-config_{context}"]
+= Configuring {product} deployment
+
+The {product} operator exposes a `rhdh.redhat.com/v1alpha2` API Version of its Custom Resource Definition (CRD). This CRD exposes a generic `spec.deployment.patch` field, which gives you full control over the {product-short} Deployment resource. This field can be a fragment of the standard `apps.Deployment` Kubernetes object.
+
+For example, the following specification fragment will:
+
+* Add an additional volume named `my-volume` and mount it under `/my/path` in the {product-short} application container.
+* Replace the default `dynamic-plugins-root` volume with a persistent volume claim (PVC) named `dynamic-plugins-root`. Note the `$patch: replace` directive, otherwise a new volume will be added.
+* Set the CPU request for the {product-short} application container to 250m.
+* Set the label `my=true` for the {product-short} Pod.
+* Add a new `my-sidecar` sidecar container into the {product-short} Pod.
+
+[NOTE]
+====
+To learn more about merging, see link:https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md#basic-patch-format[Strategic Merge Patch].
+====
```

Comment:
Is this information relevant for the reeader?

If yes, it might be a good fit for an `.Additional resources` section at the end of the procedure.